gradle/integration-test.gradle

```diff
@@ -12,11 +12,7 @@ task integrationTest(type: Test) {
   group = 'verification'
 }
 
-['**/ScanIT_Gradle_Versions_5_0*.class',
- '**/ScanIT_Gradle_Versions_5_5*.class',
- '**/ScanIT_Gradle_Versions_6_0*.class',
- '**/ScanIT_Gradle_Versions_6_6*.class',
- '**/ScanIT_Gradle_Versions_7_0*.class'].eachWithIndex { className, index ->
```

Comment:
Had to delete these tests as mentioned in this [comment](https://github.com/sonatype-nexus-community/scan-gradle-plugin/issues/146#issue-2053155602) "_We'd need to stop supporting Gradle versions lower than 7.6.2_".